### DISCUSSION TITLE
Vike + ReactTS running on Fastly

### DISCUSSION THREAD

**Discussion by tampa-cristian** [vikejs/vike#1801] [Discussion]
Hello everyone, I'm trying to deploy my app (Vike + React TS) to [Fastly](https://www.fastly.com), yet since the build generates two folders, client/server, I'm not yet sure how to do it. What should be the entry point in the build `dist/`? Others frameworks like NextJs have a special faslty compiler -> [faslty-nextjs-compute](https://github.com/fastly/next-compute-js). How should the Vike integrate with such platform? Anyone has done it before / got any tips and tricks on how to achieve this?

**Comment by brillout** [comment]
I ain't that familiar with Fastly so I can't help much on that side, but let us know if you need something from Vike's side.

Also, if you can create a discussion on Fastly's side and link to this discussion that'd be great.
